### PR TITLE
[16:26] Генеральныйส็็็็็็็็็็็Секретарь: You may as well remove the speed potions because they do literally nothing

### PR DESCRIPTION
**Pull request by Amelia0010**
### Intent of your Pull Request

implements monster's fantastic idea!

#### Changelog

:cl:  
rscdel: you can no longer get speed potions :)
/:cl:

### CODE REVIEW

File: code/modules/reagents/chemistry/recipes/slime_extracts.dm [hunk 1]
```diff
@@ -193,9 +193,9 @@
 	required_container = /obj/item/slime_extract/blue
 	required_other = TRUE
 
-/datum/chemical_reaction/slime/slimestabilizer/on_reaction(datum/reagents/holder)
+/* Yogs /datum/chemical_reaction/slime/slimestabilizer/on_reaction(datum/reagents/holder)
 	new /obj/item/slimepotion/slime/stabilizer(get_turf(holder.my_atom))
-	..()
+	..() Yogs */
```
Comment (monster860) [inline code/modules/reagents/chemistry/recipes/slime_extracts.dm]: dude.

this doesnt even remove the speed potion. it removes a completely unrelated item

Comment (Amelia0010) [reply code/modules/reagents/chemistry/recipes/slime_extracts.dm]: fixed, it now does what you wanted 👍

**Comment by k-k-karma** [comment]
not modular

**Comment by Amelia0010** [comment]
ok

**Pull request merged**